lib/components/BuilderUI.jsx

```diff
@@ -251,38 +252,74 @@ const createPools = async (
   }
 }
 
+const FORM_FIELD_DEFAULTS = {
+  resultingContractAddresses: {},
+  depositToken: {},
+  prizePool: {},
+  cToken: '',
+  stakedTokenData: undefined,
+  stakedTokenAddress: '',
+  yieldSourceData: undefined,
+  yieldSourceAddress: '',
+  rngService: '',
```

Comment:
Should we default this to blockhash for all? That's also the only input other than the deposit token that's required right now and doesn't have a default value. It's pretty sweet being able to just select the token and press create prize pool.

Reply:
I like the idea behind that but I think it's better to make people think about it. I'd actually like to go the opposite direction and do a better job of telling them about how blockhash isn't as safe.